modules/migration-about-configuring-proxies.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/installing-3-4.adoc
+// * migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc
+// * migration_toolkit_for_containers/installing-mtc.adoc
+// * migration_toolkit_for_containers/installing-mtc-restricted.adoc
+
+:_content-type: CONCEPT
+[id="migration-about-configuring-proxies-{context}"]
+= Proxy configuration
+
+For {product-title} 4.1 and earlier versions, you must configure proxies in the `MigrationController` custom resource (CR) manifest after you install the {mtc-full} Operator because these versions do not support a cluster-wide `proxy` object.
+
+For {product-title} 4.2 to {product-version}, the {mtc-full} ({mtc-short}) inherits the cluster-wide proxy settings. You can change the proxy parameters if you want to override the cluster-wide proxy settings.
+
+== Direct volume migration
+
+Direct Volume Migration (DVM) was introduced in MTC 1.4.2. DVM supports only one proxy. The source cluster cannot access the route of the target cluster if the target cluster is also behind a proxy.
+
+If you want to perform a DVM from a source cluster behind a proxy, you must configure a TCP proxy that works at the transport layer and forwards the SSL connections transparently without decrypting and re-encrypting them with their own SSL certificates. A Stunnel proxy is an example of such a proxy.
+
+=== TCP proxy setup for DVM
+
+You can set up direct connection between the source and the target cluster through a TCP proxy and configure the `stunnel_tcp_proxy` variable in the `MigrationController` CR use the proxy:
+
+[source, yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigrationController
+metadata:
+  name: migration-controller
+  namespace: openshift-migration
+spec:
+  [...]
+  stunnel_tcp_proxy: http://username:password@ip:port
+----
+
+Direct volume migration (DVM) only supports Basic Authentication for the proxy. Moreover, DVM only works behind proxies which can tunnel a TCP connection transparently. HTTP/HTTPS proxies in man-in-the-middle mode will not work. The existing cluster-wide proxies may or may not support this behavior. As a result, the proxy settings for DVM are intentionally kept different from usual proxy configuration in {mtc-short}.
```

Comment:
Basic Authentication >> basic authentication
Add **the** to the line  .....kept different from **the** usual proxy ....